js/tests/unit/dropdown.spec.js

```diff
@@ -1102,6 +1102,47 @@ describe('Dropdown', () => {
       dropdown.show()
     })
 
+    it('should not collapse the dropdown when clicking an item in navbar dropdown', done => {
+      fixtureEl.innerHTML = [
+        '<nav class="navbar navbar-expand-md navbar-light bg-light">',
```

Comment:
I mean you should not use this navbar here (It is not needed). Since the behavior you are testing is not specific to the navbar only.

Reply:
You are right. I mean I couldn't even find an example that uses the dropdown like that =/ I basically used the code like it was in the issue.

Reply:
Ok nvm my fault.. Been already some days since I did this issue..

Reply:
I didn't even look to much into that problem as it was quite easy to fix..

Reply:
Took me longer to write the test for this.. and I guess `clearMenus()` spy isn't called because the original is bound by reference on the document..